tests/queries/0_stateless/01643_replicated_merge_tree_fsync_smoke.sql

```diff
@@ -1,5 +1,6 @@
--- Tags: no-parallel
+-- Tags: no-parallel, no-s3-storage
 -- no-parallel -- for flaky check and to avoid "Removing leftovers from table" (for other tables)
+-- no-s3-storage -- hangs now, need investigation
```

Comment:
Actually this test never passed with S3 configuration on CI.
This hasn't been noticed because in the PR that adds this test #38993, because there was no report not from github actions for S3 configuration since github action job failed.
Should be fixed in #39214